tests/components/application-connector/README.md

```diff
@@ -168,3 +168,10 @@ To run the tests without removing all the created resources afterwards, run:
 ``` shell
 make test-gateway-debug
 ```
+
+   > **CAUTION:** In this mode, you must remember to enable the sidecars as soon as all the tests have been completed:
+   > ```bash
+   > make enable-sidecar-after-mtls-test
+   > ```
+   > Keep in mind that `completed` does not mean `succeeded`. You must do it even if the tests fail. 
```

Comment:
``````suggestion
> **CAUTION:** In this mode, you must remember to enable the sidecars as soon as all the tests have been completed:
> ```bash
> make enable-sidecar-after-mtls-test
> ```
> Keep in mind that `completed` does not mean `succeeded`. You must do it even if the tests fail. 
``````

This indentation is not needed. It was here, because previously it was under a list element.